apollo/src/main/graphql/com/hedvig/android/owldroid/graphql/home.graphql

```diff
@@ -73,4 +73,23 @@ query HomeQuery($locale: Locale!, $languageCode: String!) {
     }
     body
   }
+
+  quoteBundle(input: $quoteBundleInput) {
```

Comment:
Does this query gracefully handle being passed an empty list of `ID`s? What about an `ID` that does not exist, will it throw errors? If yes, how do we deal with that error?

Reply:
You are right, this is not handled very well. At the moment the whole query fails and we get an error message saying we need to supply at least one id. Should we do this query separate from the home query? 

I suppose what we really want is for the query to look something like 
```
Query.quoteBundle(input: QuoteBundleInput!): QuoteBundleResult!
```

Where a `QuoteBundleResult` is a union type and can have values `QuoteBundle` or `EmptyBundle`?